.travis.yml

```diff
@@ -49,8 +50,12 @@ matrix:
 # 7.2
     - php: 7.2
       env: TEST_CONFIG="phpunit.xml"
-    - php: 7.2
-      env: ELASTICSEARCH_VERSION="1.4.2" TEST_CONFIG="phpunit-integration-legacy-elasticsearch.xml" SYMFONY_VERSION="~2.8.46"
+# 7.3
+    # Temp: Need to use --ignore-platform-reqs due to: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3697
+    - php: 7.3
+      env: TEST_CONFIG="phpunit.xml" COMPOSER_FLAGS="--ignore-platform-reqs"
```

Comment:
Just to be sure: no need to specify `SYMFONY_VERSION` here? Don't know which version should have priority here (3.x or 2.x).

Reply:
I'll update this

Reply:
@mnocon Ok with the change in regards to symfony version?

Reply:
@andrerom if I understand correctly, this job: https://github.com/ezsystems/ezpublish-kernel/pull/2504/files#diff-354f30a63fb0907d4ad57269548329e3L45 will be now running with Symfony 2.x instead of 3.x? (I have no idea which job should be running with which Symfony version)

If this is intended then sure, looks good to me (also: good idea to set 2.x globally!)

Reply:
Sure, thanks! Looks good to me then